### PR TITLE
Added more of the sklearn regressors to the presets

### PR DESCRIPTION
**Pull request by enisnazif**
Added the large majority of the more popular regressors of sklearn. I am aware that a few may be missing. Also, I tidied the code slightly and split the regressors and classifiers into two sections.

### CODE REVIEW

File: xcessiv/presets/learnersetting.py [hunk 1]
```diff
@@ -2,31 +2,44 @@
...
-    'xgboost_regressor',
+
+    # Regressors
+    'sklearn_random_forest_regressor',
+    'sklearn_extra_trees_regressor',
+    'sklearn_bagging_regressor',
+    'sklearn_GP_regressor',
+    'sklearn_ridge_regressor',
+    'sklearn_lasso_regressor',
+    'sklearn_kernelRidge_regressor',
```
Comment (reiinakano) [inline xcessiv/presets/learnersetting.py]: stylefix: probably better to use sklearn_kernel_ridge_regressor

File: xcessiv/presets/learnersource.py [hunk 2]
```diff
@@ -52,10 +77,64 @@
...
+sklearn_kernelRidge_regressor_source = \
+    """from sklearn.kernel_ridge import KernelRidge
+
+base_learner = KernelRidge(random_state=8)
+"""
+
+sklearn_knn_regressor_source = \
+    """from sklearn.neighbors import KNeighborsRegressor
+
+base_learner = KNeighborsRegressor(random_state=8)
```
Comment (reiinakano) [inline xcessiv/presets/learnersource.py]: I don't think KNN has a random_state parameter. Please also double check the other estimators to see if they have a random_state parameter.

**Comment by reiinakano** [comment]
Looks good so far! Just made a few comments that should be addressed first.

**Comment by reiinakano** [comment]
Think it looks good! Am a bit scared of pulling these without adequate unit tests, but I guess it's my fault for not writing them in the first place. I really need to get to doing those.

Thanks a lot!

**Comment by enisnazif** [comment]
no problem :) if you'd like, I can start having a look at writing these tests?

**Comment by reiinakano** [comment]
@enisnazif That would be very helpful! :)

**Pull request merged**